### ISSUE TITLE
[TF2] Change pyro's flames to look more like they used to before the Jungle Inferno update

### ISSUE THREAD

**Issue by Rongeno** [ValveSoftware/Source-1-Games#4147]
Jungle Inferno update changed a lot about pyro's flames, including how they look. And to be honest, now they look too bright, ugly and don't fit into TF2's artstyle

This is how pyro's flames look after Jungle Inferno:
![20220809202701_1](https://user-images.githubusercontent.com/108897156/183733220-b94dd937-8ad9-4859-83fc-7b02ee5dffae.jpg)

This is how they used to look before Jungle Inferno:
![ezgif-2-18d329bab8](https://user-images.githubusercontent.com/108897156/183733317-485d68c6-e69f-4b91-ab3c-6e2fa3df9c49.jpg)

And it doesn't even end there, because current pyro flames look inconsistent compared to other flame particles in the game. For example: 

Burning players:
![20220809202852_1](https://user-images.githubusercontent.com/108897156/183733586-e6882fd2-61c1-46c5-bcd4-1ef2a04eb4f3.jpg)
![20220809202756_1](https://user-images.githubusercontent.com/108897156/183733593-c9de2fd1-7e89-4673-ba77-5344a686bfa4.jpg)

Unusual effect: Burning Flames
![Unusual_Burning_Flames](https://user-images.githubusercontent.com/108897156/183733688-f34fc228-e661-42fb-837e-a620685fea5a.png)

**Comment by AzureAzel** [comment]
Yeah, it's definitely way too yellow / white now. It needs to be more orange. It also seems like the original particle had more detail where it would go from yellow to orange in almost gradient-like way. The new particle is mostly just a static color.

**Comment by condor00fr** [comment]
Prehaps taking a look at the TF2 visual fix pack, it's on Github and has a goal of fixing many issues and making the game look much better, for instance weapons now look a lot more like how they did at the game's release, they also made brand new and more optimised particles, such as pre jungle inferno flames with fixed hitbox, pre gun mettle minigun flash, and even it's own unique ones like explosions for the jumpers that fit the theme.

The pack is currently in it's last release candidate before sending it directly to Valve as a possible new uptate before starting part 2 (wich are animations and player models)

It's definitly something to consider since theirs work is really great and makes the game a lot more modern while keeping the tf2 artstyle intact:)

**Comment by FrozenDragon0** [comment]
While I do like the Jungle inferno flames, I do agree that it is quite static with it's color and not very interesting and definitely should have it's color changed to a more orange look.

**Comment by ShadowMan44** [comment]
A better option: Let the player decide which one to use in the options menu.

**Comment by torzod** [comment]
The reason the old flame particles and the current Phlogistinator particles have a smooth gradient from red to yellow is because they use blending, a method of image composition in which one layer's colors are added to the layers below it, rather than replacing them. This seems to be disabled on the modern particles for whatever reason, or they may just be close enough to white that the blending is nigh unnoticeable.

**Comment by Flpstrike** [comment]
Those flames should be team-colored on my opinion
